__test__/features/Events/updateEventHelpers/makeVevent.test.ts

```
@@ -35,7 +43,9 @@ function baseEvent(overrides: Partial<CalendarEvent> = {}): CalendarEvent {
     timezone: TZID,
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)